### ISSUE TITLE
RFC: change mapping to chain mapper

### ISSUE THREAD

**Issue by f3l1x** [nettrine/orm#27]
I would like to change internal mapping driver to `DriverChain`. It would be more suitable for separate extensions, each could register own mapper, or for modular system.

Snippet from doc: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/advanced-configuration.html#multiple-metadata-sources

```
$chain = new DriverChain(); 
$chain->addDriver($xmlDriver, 'Doctrine\Tests\Models\Company'); 
$chain->addDriver($yamlDriver, 'Doctrine\Tests\ORM\Mapping');
```

**Comment by f3l1x** [comment]
cc @uestla @jiripudil @MartkCz @aleswita

**Comment by mabar** [comment]
@f3l1x Couldn't we do same thing in Apitte?

**Comment by f3l1x** [comment]
Sure. Good idea.

**Comment by f3l1x** [comment]
Will be implemented in v0.6.
